### DISCUSSION TITLE
Add existing Python installations to Homebrew pyenv

### DISCUSSION THREAD

**Discussion by harri00413** [pyenv/pyenv#2055] [Q&A] [answered]
I have installed Python in the past with  a zip from python.org, with homebrew and whatnot. Now I wanted to start to use pyenv to try and sort this installation mess. I now see I have to install Python again.. is adding to the mess. Can't I just add the installs python3.8 and 3.9 to the pyenv managed installs?
I can't uninstall the others due to dependencies.

help

**Comment by native-api** [comment, marked as answer]
https://github.com/pyenv/pyenv/issues/1244#issuecomment-835074710

> If you have an existing Python installation under `<prefix>`, you can symlink `$(pyenv root)/versions/<arbitrary name>` to that `<prefix>`. Pyenv would see and handle this `<arbitrary name>` like any other version entry.

https://github.com/pyenv/pyenv/issues/1244#issuecomment-491229852

> 
> I just had a great experience with this [plugin](https://github.com/doloopwhile/pyenv-register).
> 

(this plugin does the same symlinking)
